oxidation/crates/parsec_client_types/src/local_manifest.rs

```diff
@@ -324,6 +324,10 @@ impl LocalFileManifest {
         reference.version = remote_manifest.version;
         reference == *remote_manifest
     }
+
+    pub fn set_single_block(&mut self, block: u64, new_chunk: Chunk) {
```

Comment:
`block: usize` is better
Is it our new convenience to use u64 everywhere ?

Reply:
I have no idea nor preference, but the team should make a decision about this and we should stick to it.

I used `u64` because that's what's used in the different structures (`Chunk.start`, `LocalFileManifest.size`, etc.)